src/classad/classad.cpp

```diff
@@ -47,15 +47,24 @@ bool _useOldClassAdSemantics = false;
 // Should parsed expressions be cached and shared between multiple ads.
 // The default is false.
 static bool doExpressionCaching = false;
+int  _expressionCacheMinStringSize = 128;
```

Comment:
Is the value 128 based on anything other than intuition? I would have picked something closer to 64.

Reply:
based on testing back in 7.9 time frame.     and envelope node is ~64 bytes, a cache entry is about 100.   the effect is pretty minor either way.